src/arithmetic/repository.c

```diff
@@ -34,17 +22,17 @@ static void __agg_initRegistry() {
 void Agg_RegisterFunc(const char *name, AggFuncInit f) {
 	__agg_initRegistry();
 	char lower_func_name[32] = {0};
```

Comment:
This is a good opportunity to remove this initialisation
char lower_func_name[32];
Please apply throughout the modified files.